spring-boot-autoconfigure/src/test/java/org/axonframework/springboot/AxonAutoConfigurationWithMicrometerMetricsTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.springboot;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import org.axonframework.micrometer.GlobalMetricRegistry;
+import org.axonframework.micrometer.MetricsConfigurerModule;
+import org.axonframework.springboot.autoconfig.MetricsAutoConfiguration;
+import org.junit.*;
+import org.junit.runner.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
+import org.springframework.boot.autoconfigure.jmx.JmxAutoConfiguration;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
+import org.springframework.boot.autoconfigure.web.reactive.function.client.WebClientAutoConfiguration;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.*;
+
+@ContextConfiguration
+@EnableAutoConfiguration(exclude = {
+        JmxAutoConfiguration.class, WebClientAutoConfiguration.class, HibernateJpaAutoConfiguration.class,
+        DataSourceAutoConfiguration.class, MetricsAutoConfiguration.class
```

Comment:
By setting the `@AutoConfigureAfter` on the `MetricsAutoConfiguration`, I'd assume the exclusion of `MetricsAutoConfiguration` in this test wouldn't be necessary too :thinking: